test/conftest.py

```diff
@@ -272,6 +279,29 @@ def ferc1_xbrl_taxonomy_metadata(ferc1_engine_xbrl: sa.Engine):
     return result.output_for_node("raw_ferc1_xbrl__metadata_json")
 
 
+def _pudl_etl_job_factory(
```

Comment:
I removed one layer of function nesting here because it seemed unnecessary based on the [reconstructable jobs docs](https://docs.dagster.io/_apidocs/jobs#reconstructable-jobs).

Since we're only using `execute_in_process` right now anyways, we don't even need a reconstructable job, but it seems fine to leave this because the logic is pretty straightforward now.